src/olympia/blocklist/tests/test_admin.py

```diff
@@ -1801,6 +1802,7 @@ def setUp(self):
         )
         self.change_url = reverse('admin:blocklist_block_change', args=(self.block.pk,))
         self.submission_url = reverse('admin:blocklist_blocklistsubmission_add')
+        self.task_user = user_factory(id=settings.TASK_USER_ID)
```

Comment:
Why is this needed here?

Reply:
A few tests in that class call code that eventually passes by `disable_addon_for_block`, for example: https://github.com/mozilla/addons-server/blob/72d5bbaba243c88b13bad9e3d2bf410ec8134054/src/olympia/blocklist/tests/test_admin.py#L1895:L1895

Reply:
Interesting. I guess you don't need `self.task_user`?
```suggestion
        # We need the task user because some test cases eventually trigger
        # `disable_addon_for_block()`.
        user_factory(id=settings.TASK_USER_ID)
```

Reply:
Ok, thanks